participant/tests/test_api.py

```diff
@@ -292,5 +292,14 @@ def test_participant_scores_debrief_behaviour_responses(self):
                 )
                 self.assertEqual(
                     behaviour,
-                    response.data["phishing_emails"][0]["participant_behaviour"],
+                    response.data["phishing_emails"][0]["participant_behaviour"][
+                        "behaviour"
+                    ],
                 )
+                if behaviour == "neutral":
+                    self.assertEqual(
```

Comment:
- let's use `assertIsNone` instead for `neutral`
- for anything else let's include a check to ensure `action_id` in the response matches with our `action.id` (L281)